Extensions/servant.js

```diff
@@ -1140,16 +1140,13 @@ XKit.extensions.servant = new Object({
 					}
 				}
 
-				if (m_post !== "") {
-					var to_run = function(post) {
-						eval(parameter_fixed + "\n//# sourceURL=xkit/servant/servant" + (new Date()).getTime() + ".js");
-					};
-
-					to_run(m_post);
-				} else {
-					eval(parameter_fixed + "\n//# sourceURL=xkit/servant/servant" + (new Date()).getTime() + ".js");
+				var post = null;
```

Comment:
is this necessary? i feel like `post` not being defined at all in non-post-affecting javascript actions would be more expected

Reply:
@aliadnan1984pk this still needs to be resolved before merging. I know it's a tiny issue but expected behaviour is important here I think.

(I'll make the change myself and merge in a week or so if I hear nothing from you)